wnfs-namefilter/src/lib.rs

```diff
@@ -1,5 +1,6 @@
+//! TODO(appcypher): Needs documentation.
```

Comment:
Same Q: Should this happen this PR?
I can see "no" being fine, given that we may swap this out for name accumulators soon (TM)

Reply:
```suggestion
//! WNFS Namefilter
//------------------------------------------------------------------------
// Documentation:
//------------------------------------------------------------------------
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```